app/src/editor/shared/components/modules/Chart.jsx

```diff
@@ -107,9 +135,8 @@ const ChartModule2 = (props) => {
                 ref={subscriptionRef}
             />
             <Chart
-                datapoints={seriesData}
                 options={module.options || {}}
-                series={series}
+                callback={setChart}
```

Comment:
When does this callback? `callback` maybe could have a more self-explanatory, less generic name?